src/Microsoft.Health.Fhir.Shared.Api/Features/Filters/OperationOutcomeExceptionFilterAttribute.cs

```diff
@@ -116,6 +117,7 @@ public override void OnActionExecuted(ActionExecutedContext context)
                         break;
                     case BadRequestException _:
                     case RequestNotValidException _:
+                    case ServerSideRequestForgeryException _:
```

Comment:
Maybe semantic interpretation, but I'd suggest 403: Forbidden over 400. 
Why: 403 is rejection based on untrusted origins.
400: Was provided in an invalid format and was not processable.
Open to thoughts?

Reply:
Makes sense, updated